Assets/Scripts/Towers/Specific towers/Lightning turret/LightningShootable.cs

```diff
@@ -19,7 +19,7 @@ public class LightningShootable : MonoBehaviour, TurretInterface
     public GameObject drainRay;
 
     //LighntingEffect duration
-    public float effectDuration = 0.3f;
+    public float effectDuration = 0.7f;
```

Comment:
Was this changed on purpose? Because it's not changed in the prefab (`LightningTurret.prefab` - more specifically the `LowPolyLightning` game object) 🤔